apps/s_client.c

```diff
@@ -3245,6 +3245,12 @@ static void print_stuff(BIO *bio, SSL *s, int full)
     BIO_printf(bio, "Expansion: %s\n",
                expansion ? SSL_COMP_get_name(expansion) : "NONE");
 #endif
+#ifndef OPENSSL_NO_KTLS
+    if (BIO_get_ktls(SSL_get_wbio(s)))
+        BIO_printf(bio_err, "Using Kernel TLS for sending\n");
+    if (BIO_get_ktls(SSL_get_rbio(s)))
+        BIO_printf(bio_err, "Using Kernel TLS for receiving\n");
```

Comment:
Actually - that raises an API question. Do we actually need something like `BIO_get_ktls_recv()` and `BIO_get_ktls_send()` or similar? Or (perhaps) just the latter for now since we don't (yet) support the former.

Reply:
We do need it for both directions eventually. I think that my next patchset that introduces receive support for kTLS in OpenSSL should add that. I'll remove the receive side print, because it doesn't make any sense for the code at the moment.